include/systems/parameter_multiaccessor.h

```diff
@@ -75,7 +75,8 @@ class ParameterMultiAccessor : public ParameterAccessor<T>
   {
     libmesh_assert(!_accessors.empty());
 #ifndef NDEBUG
-    T& val = _accessors[0]->get();
+    // Compare other values to the last one we'll change
+    T& val = _accessors[_accessors.size()-1]->get();
```

Comment:
`_accessors` is a vector, so `.back()`?  I haven't analyzed the rest of the code, but this seems dangerous if there's any chance that `_accessors.size()==0`...